src/extensions/menu.extension.ts

```diff
@@ -81,22 +97,25 @@ export class Menu {
     }
 
     switch (commandName) {
-      case '[ back ]': {
-        await this.showMenu(parentCommands.substr(0, parentCommands.lastIndexOf(' ')));
+      case messages.back: {
+        await this.showMenu(
+          parentCommands.substr(0, parentCommands.lastIndexOf(' ')),
+          options
```

Comment:
At this point the current options are moved up. This means that the options of the parent menu are not applied, but the options of the children menu are used.

We have two options here:
1. all menus are the same. Then the wrong options are not noticeable, but then the configuration should be done by the options in one place so that they don't have to be specified in every menu, or
2. each menu should be able to be configured individually, but then instead of calling the menu directly, we have to execute the respective parent command, which could look like this:

```
case messages.back: {
        // Get command
        let command = commands.filter(
          (c: any) =>
            c.commandPath.join(' ') === parentCommands
              .trim().replace(/\s\(.*\)$/, '')
              .split(' ').slice(0,-1).join(' ')
        )[0];
        if (!command) {
          command = commands[0];
        }

        // Run command
        try {
          this.toolbox.parameters.options.fromGluegunMenu = true;
          await command.run(this.toolbox);
          process.exit();
        } catch (e) {
          // Abort via CTRL-C
          if (!e) {
            console.log(messages.bye);
          } else {
            // Throw error
            throw e;
          }
        }
        break;
      }
```

Reply:
Thanks for this!
I implemented the code you suggested & also added a small "cache" which would allow re-using the options from the "main menu"